### PR TITLE
Ops 1350 bli list page

### PR DESCRIPTION
**Pull request by johndeange**
## What changed

Add initial BLI List page.

## Issue

https://github.com/HHS/OPRE-OPS/issues/1350

## How to test

Click on `Budget Lines` on the homepage to navigate to the BLI Page.

## Screenshots

image

### CODE REVIEW

File: frontend/src/index.jsx [hunk 1]
```diff
@@ -139,7 +140,7 @@ const router = createBrowserRouter(
                 />
                 {/*/!* Default to BudgetAndFunding *!/*/}
                 {/*<Route exact path="" element={<Navigate to={"budget-and-funding"} />} />*/}
-                <Route path="budget-lines" element={<Agreement />} />
+                <Route path="/budget-lines" element={<BudgetLineItemList />} />
```
Comment (fpigeonjr) [inline frontend/src/index.jsx:143]: 😄 wonder if we should setup redirects to add the all blis filter params?

so if you go to `/budget-lines` it should route you to `/budget-lines?filter=all-budget-line-items`?

File: frontend/src/pages/budgetLines/list/BudgetLineItemList.jsx [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+        const myBLIs = filteredBudgetLineItems.filter(() => {
+            return true;
+        });
+        sortedBLIs = sortBLIs(myBLIs);
+    } else {
+        // all-budget-line-items
+        sortedBLIs = sortBLIs(filteredBudgetLineItems);
+    }
+
+    console.log("filters", filters);
```
Comment (fpigeonjr) [inline frontend/src/pages/budgetLines/list/BudgetLineItemList.jsx:56]: these will be helpful when we start filter work

File: frontend/src/components/UI/Header/Menu.jsx [hunk 3]
```diff
@@ -26,6 +26,9 @@ export const Menu = () => {
                 <li className="usa-nav__primary-item">
                     <Link to="/agreements?filter=all-agreements">Agreements</Link>
                 </li>
+                <li className="usa-nav__primary-item">
+                    <Link to="/budget-lines?filter=all-budget-line-items">Budget Lines</Link>
```
Comment (fpigeonjr) [inline frontend/src/components/UI/Header/Menu.jsx:30]: maybe keep this `/budget-lines` and let `react-router` redirect to the filters?

**Review by fpigeonjr — APPROVED** [review]
pulled locally and looks great

File: frontend/src/components/UI/Header/Menu.jsx (hunk 3 above)
Comment (fpigeonjr) [reply frontend/src/components/UI/Header/Menu.jsx:30]: oh and this will light up the active tab...

Comment (tdonaworth) [reply frontend/src/components/UI/Header/Menu.jsx:30]: Did we want start by pointing them to a narrower view first? ie: Main link would take them to "My Budgetlines" ?

Comment (fpigeonjr) [reply frontend/src/components/UI/Header/Menu.jsx:30]: Not sure where we landed on this but there was some discussion on the topic. Right now it is set to all BLIs.

**Review by tdonaworth — APPROVED** [review]

**Pull request merged**